packages/flutter/lib/src/material/button_theme.dart

```diff
@@ -0,0 +1,263 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/widgets.dart';
+
+import 'theme.dart';
+
+/// Used with [ButtonTheme] and [ButtonThemeData] to define a button's base
+/// colors, and the defaults for the button's minimum size, internal padding,
+/// and shape.
+///
+/// See also:
+///
+///  * [RaisedButton], which styles itself based on the ambient [ButtonTheme].
+///  * [FlatButton], which styles itself based on the ambient [ButtonTheme].
+enum ButtonTextTheme {
+  /// Button text is black or white depending on [ThemeData.brightness].
+  ///
+  /// For more information see [FlatButton], [RaisedButton].
+  normal,
+
+  /// Button text is [ThemeData.accentColor].
+  ///
+  /// For more information see [FlatButton], [RaisedButton].
+  accent,
+
+  /// A button's text color is based on [ThemeData.primaryColor].
+  ///
+  /// For more information see [FlatButton], [RaisedButton].
+  primary,
```

Comment:
the way enums appear in dartdocs, it's all on one page, so i'd omit the redundant "For more information see [FlatButton], [RaisedButton]." since you already have it in the enum-level docs.